sympy/physics/mechanics/pathway.py

```diff
@@ -178,7 +183,7 @@ class LinearPathway(PathwayBase):
     ==========
 
     attachments : tuple[Point, Point]
```

Comment:
Would "ends" or "endpoints" be a better word and more generic?

Reply:
endpoint
ĕnd′point″
noun

    Either of two points marking the end of a line segment.